lib/src/configuration_wire_protocol.dart

```diff
@@ -62,9 +62,6 @@ class ObfuscatedPrecomputedConfigurationResponse {
 
 /// Represents a precomputed flag with a value and type
 class ObfuscatedPrecomputedFlag {
-  /// Optional flag key
-  final MD5String? flagKey;
```

Comment:
this was erroneously added - the `flagKey` is not present in the payload, it is the map key